basic/http/collect-user.test.js

```diff
@@ -25,5 +28,26 @@ describe('app', () => {
     const response = await request(server).get('/users');
     expect(Object.keys(response.body.data[0])).toStrictEqual(["id","name"]);    
   });
+
+  it('should add another object with a password', async () => {
+    const name = chance.name(),
+    id = chance.natural();
+  
+    const response = await request(server).post('/users').send({name, id});
+    expect(response.body.data[response.body.data.length-1])
+      .toStrictEqual({name, id});
+  });
+
+  it.only('should update existing object', async() =>{  
+
+      const fetchResponse = await request(server).get(`/users`);
+      const firstUser = fetchResponse.body.data[0];
+      const updateResponse = await request(server).patch(`/users/${firstUser.id}`).send({name: 'hello'});
+
+      expect(updateResponse.body.data[0].name)
+      .toStrictEqual(firstUser.name);
```

Comment:
You did not even update the firstUser's name and the assertion is wrong. You should assert if the name is equals to `'hello'`